hudi-common/src/main/java/org/apache/hudi/common/table/view/RocksDbBasedFileSystemView.java

```diff
@@ -371,6 +371,47 @@ void removeBootstrapBaseFileMapping(Stream<BootstrapBaseFileMapping> bootstrapBa
         schemaHelper.getPrefixForSliceViewByPartitionFile(partitionPath, fileId)).map(Pair::getValue)).findFirst());
   }
 
+  @Override
+  protected void resetReplacedFileGroups(final Map<HoodieFileGroupId, HoodieInstant> replacedFileGroups) {
```

Comment:
@bvaradar if you can take a pass at these, that would be great

Reply:
General question I have around incremental file system view and rocksDB like persistent file system view storage is whether we will keep this list updated. i.e when the archival/cleaning runs, how do we ensure the deleted replaced file groups are no longer tracked inside rocksdb. 

I guess the lines below, are doing a bulk delete and insert to achieve the same?

Reply:
Yes,  we do delete and insert. Definitely, would be helpful to have someone with more experience review this.

Reply:
This part looks good. Going through rest of the code